src/sentry/sentry_metrics/indexer/postgres/postgres_v2.py

```diff
@@ -253,12 +258,16 @@ def resolve(self, use_case_id: UseCaseID, org_id: int, string: str) -> int | Non
         table = self._get_table_from_metric_path_key(metric_path_key)
         try:
             if metric_path_key is UseCaseKey.PERFORMANCE:
+                assert issubclass(table, PerfStringIndexer), table
```

Comment:
I wanted to do `assert table is PerfStringIndexer` since that's what it has to be -- but mypy doesn't understand that narrowing unfortunately!